src/runtime/pkg/device/config/config.go

```diff
@@ -81,6 +81,9 @@ const (
 
 	// VirtioFSNydus means use nydus for the shared file system
 	VirtioFSNydus = "virtio-fs-nydus"
+
+	// VirtioFSNone means files will be copied into the guest system
+	VirtioFSNone = "none"
```

Comment:
I am fine with naming this as none, but was wondering if making as this "copy-files" instead of  "none" would be benefecial in terms of adding clarity.